src/commands/createNewProject/ScriptProjectCreatorBase.ts

```diff
@@ -17,7 +17,6 @@ csx
 .vs
 edge
 Publish
-.vscode
```

Comment:
Consider entering a bug against func init as well?  This definitely seems wrong.

Reply:
We have an existing bug to improve the relationship between the func cli and our extension. I added a comment to that issue to make sure we don't forget

Reply:
https://github.com/Microsoft/vscode-azurefunctions/issues/203